modules/light-clients/11-beefy/types/update_test.go

```diff
@@ -5,8 +5,8 @@ import (
 	"encoding/binary"
 	"encoding/hex"
 	"fmt"
-	types2 "github.com/cosmos/cosmos-sdk/x/params/types"
-	"github.com/cosmos/ibc-go/v3/modules/core/02-client/keeper"
+	// types2 "github.com/cosmos/cosmos-sdk/x/params/types"
```

Comment:
can't they be removed?

Reply:
nop, they are needed for the updateState call further down. So I uncommented it and made the checks optional.